### ISSUE TITLE
Rollbar is using URI.escape that is already obsolete in ruby v2.7

### ISSUE THREAD

**Issue by pangtongs** [rollbar/rollbar-gem#989]
Rollbar is using URI.escape that is already obsolete in ruby v2.7
and giving me some noise in error debugging
```
/usr/local/bundle/gems/rollbar-2.26.0/lib/rollbar/scrubbers/url.rb:73: warning: URI.escape is obsolete
```

**Comment by waltjones** [comment]
@pangtongs Thank you for the report. Will get this updated soon.

**Comment by waltjones** [comment]
Released in v2.27.1
